salt/profiles/config/etc-nginx-sites-enabled-profiles.conf

```diff
@@ -16,6 +16,7 @@ server {
         uwsgi_read_timeout 15s;
         include /etc/uwsgi/params;
         uwsgi_param HTTP_HOST {{ pillar.profiles.default_host }};
+        uwsgi_param UWSGI_SCHEME $scheme;
```

Comment:
if this is ever load balanced, will need to be rewritten, but no problem for now